.github/workflows/pr.yml

```diff
@@ -31,6 +31,9 @@ jobs:
       if: github.event_name == 'schedule'
       run: |
         echo "::set-env name=TEST_ARGS::-count 300 -race -p 10 -failfast -timeout 240m"
+    - name: set default pr params
```

Comment:
Don't this override previous step if `github.event_name == 'schedule' `?

Reply:
Thanks! fixed.